[16.0][ADD] helpdesk_mgmt_stock

rousseldenis opened the pull request.

rousseldenis committed:
[16.0][ADD] helpdesk_mgmt_stock

rousseldenis committed:
[16.0][ADD] helpdesk_mgmt_stock

rousseldenis committed:
[IMP] helpdesk_mgmt_stock: Check user is allowed to create a ticket

rousseldenis committed:
[IMP] helpdesk_mgmt_stock: Check user is allowed to create a ticket

nicolas-delbovier-acsone committed:
[IMP] helpdesk_mgmt_stock: add location on a ticket motive.

nicolas-delbovier-acsone committed:
[IMP] helpdesk_mgmt_stock: add location on a ticket motive.

nicolas-delbovier-acsone committed:
[IMP] helpdesk_mgmt_stock: add lot reference on the helpdesk ticket

nicolas-delbovier-acsone committed:
[IMP] helpdesk_mgmt_stock: add lot reference on the helpdesk ticket

nicolas-delbovier-acsone committed:
[FIX] helpdesk_mgmt_stock: missing labels in form view.

Labels went missing in form view due to a new group being created inside an already existing group (odoo only renders the labels for leaf groups).

nicolas-delbovier-acsone committed:
[FIX] helpdesk_mgmt_stock: missing labels in form view.

Labels went missing in form view due to a new group being created inside an already existing group (odoo only renders the labels for leaf groups).

nicolas-delbovier-acsone committed:
[IMP] helpdesk_mgmt_stock: add hepldesk motive in ticket creation wizard.

nicolas-delbovier-acsone committed:
[IMP] helpdesk_mgmt_stock: add hepldesk motive in ticket creation wizard.

nicolas-delbovier-acsone committed:
[REF] helpdesk_mgmt_stock: rename function for clarity.

Add the "action" prefix to the method called by the wizard "Create Ticket" button in order to signify more clearly that this returns an action.

nicolas-delbovier-acsone committed:
[REF] helpdesk_mgmt_stock: rename function for clarity.

Add the "action" prefix to the method called by the wizard "Create Ticket" button in order to signify more clearly that this returns an action.

nihelgabsi-acsone reviewed (approved):
Functional test + code review

nicolas-delbovier-acsone committed:
[IMP] helpdesk_mgmt_stock: pre-fill stock info on ticket creation

Pass default picking/move id in context of the different tickets views opened in order to allow to pre-populate those fields when the user creates a new ticket from there.

nicolas-delbovier-acsone committed:
[IMP] helpdesk_mgmt_stock: pre-fill stock info on ticket creation

Pass default picking/move id in context of the different tickets views opened in order to allow to pre-populate those fields when the user creates a new ticket from there.

lmignon commented:
@rousseldenis Can you rebase or update the repo template to fix the pre-commit? Is your PR still draft?

nicolas-delbovier-acsone commented:
@rousseldenis Can you open this?

nicolas-delbovier-acsone committed:
[IMP] helpdesk_mgmt_stock: run pre-commit

rousseldenis commented:
/ocabot merge nobump

rousseldenis commented:
@OCA/helpdesk-maintainers

pedrobaeza commented:
/ocabot merge nobump

pedrobaeza commented:
@sbidoul please reserve the name on PyPi

sbidoul commented:
Name reserved.

pedrobaeza commented:
/ocabot merge nobump
